cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -24,6 +24,7 @@
 import com.cloudant.client.org.lightcouch.CouchDbProperties;
 import com.cloudant.client.org.lightcouch.internal.CouchDbUtil;
 import com.cloudant.http.HttpConnectionInterceptor;
+import com.cloudant.http.HttpConnectionInterceptorContext;
```

Comment:
Is this needed?

Reply:
Removed in a1c95fad4ccf0e845fe4649613fc184c33168220